src/oogeso/core/networks/edge.py

```diff
@@ -128,3 +133,11 @@ def _loss_function_constraint(self, i: int, pyomo_model: pyo.Model, piecewise_re
             f_rule=pw_y,
         )
         return constr_penalty
+
+
+class FluidEdge(Edge):
```

Comment:
Is this usefull somehow? Ref. expecting loss function above. Maybe we should have a family of Edges inheriting from Edge?

Reply:
Investigate further...